docs/presto-admin-commands.rst

```diff
@@ -506,7 +506,7 @@ Example
 -------
 ::
 
-    sudo ./presto-admin server upgrade new-rpm.rpm /tmp/cluster-configuration
+    sudo ./presto-admin server upgrade /path/to/new-rpm.rpm /tmp/cluster-configuration
```

Comment:
This should be corresponding to however we describe it here: https://github.com/prestodb/presto-admin/pull/182/files#r66441884.
If we describe it as absolute path, it's fine how's it now. If we are going to refer to it as a relative path, it should be relative in the example too.